resources/icons/16px/data-connection.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>bp3-data-connection</title>
+    <defs>
```

Comment:
please re-export these icons with simpler SVG definitions similar to the other icons in the repo. see my comment here https://github.com/palantir/blueprint/pull/4132#discussion_r427577589